src/js/vendor/buffer/utf8ToBytes.ts

```diff
@@ -9,81 +9,74 @@
  * License: MIT (https://github.com/feross/buffer)
```

Comment:
Just a nit, but this file has eslint disabled on the first line `/* eslint-disable */`
I'm surprised that it's re formated here.